public/index.html

```diff
@@ -24,6 +24,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;500;600;700&display=swap');
```

Comment:
The font in the design is Montserrat. Why did you import these font?

Reply:
We don't have to copy it Verbatim

Reply:
We don't need to use it here, if you don' t want to. Just as general info - usually that's what a Screendesign is about - that the Developer uses the given information and styling to use it in the project. In real live these are the things witch are committed by the business client or based in the company CI.

Reply:
I mean, Vasilis in this case said we don'T have to use what he used.